paths/graphql/outline.md

```diff
@@ -0,0 +1,24 @@
+---
+
+---
+
+Top-level numbers individual pages?
+
+1. Introduction
+  - Motivation, present the problem that needs solving
```

Comment:
Based on my current understanding of what this class will do, a student will be able to:
- Find, search, and interact with the docs of GraphQL
- Create a query on GraphQL to search their own contributions in the last week
- Read and interpret the response from the query
- (Maybe) Create a mutation as an issue comment based on the information from the query's response

@beardofedu is this what you're looking for?